src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchArgs.cs

```diff
@@ -40,7 +44,53 @@ public int VerifyInt(string number, int defaultValue)
                 return verifiedNumber;
             }
 
-            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalid_number, number));
+            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalidOptionValue, number, option));
+        }
+
+        public bool VerifyFormat(string format)
+        {
+            if (!string.IsNullOrEmpty(format))
+            {
+                if (string.Equals(format, "json", StringComparison.CurrentCultureIgnoreCase))
+                {
+                    return true;
+                }
+                else if (string.Equals(format, "table", StringComparison.CurrentCultureIgnoreCase))
+                {
+                    return false;
+                }
+                else
+                {
+                    throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalidOptionValue, format, "--format"));
+                }
+            }
+
+            return false;
+        }
+
+        private PackageSearchVerbosity VerifyVerbosity(string verbosity)
```

Comment:
How about remaining `VerifyVerbosity` to `GetVerbosityFromOption` to better reflect its functionality. The new name clearly indicates that the method retrieves a verbosity level based on an option provided.